packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -24,6 +24,8 @@ import { ISystemConfig } from "src/L1/interfaces/ISystemConfig.sol";
 import { IResourceMetering } from "src/L1/interfaces/IResourceMetering.sol";
 import { ISuperchainConfig } from "src/L1/interfaces/ISuperchainConfig.sol";
 import { IL1Block } from "src/L2/interfaces/IL1Block.sol";
+import { IEthBalanceWithdrawer } from "src/L1/interfaces/winddown/IEthBalanceWithdrawer.sol";
```

Comment:
This interface can be inherited here, the only thing to take into account is that it also has to be added to its interface also.